server/utils/__tests__/is-api-request.spec.js

```diff
@@ -0,0 +1,67 @@
+const isApiRequest = require("../is-api-request");
+
+describe('isApiRequest', () => {
+  it('should return `true` if route info type is `content-api`', () => {
+    const ctx = {
+      state: {
+        route: {
+          info: {
+            type: 'content-api',
+          },
+        },
+      },
+    };
+    const result = isApiRequest(ctx);
+    expect(result).toBe(true);
+  });
+
+  it('should return `false` if route info type is not `content-api`', () => {
+    const ctx = {
+      state: {
+        route: {
+          info: {
+            type: 'other-api',
+          },
+        },
+      },
+    };
+    const result = isApiRequest(ctx);
+    expect(result).toBe(false);
+  });
+
+  it('should return `false` if route info type is undefined', () => {
+    const ctx = {
+      state: {
+        route: {
+          info: {},
+        },
+      },
+    };
+    const result = isApiRequest(ctx);
+    expect(result).toBe(false);
+  });
+
+  it('should return `false` if route info is undefined', () => {
+    const ctx = {
+      state: {
+        route: {},
+      },
+    };
+    const result = isApiRequest(ctx);
+    expect(result).toBe(false);
+  });
+
+  it('should return `false` if route is undefined', () => {
+    const ctx = {
+      state: {},
+    };
+    const result = isApiRequest(ctx);
+    expect(result).toBe(false);
+  });
+
+  it('should return `false` if ctx state is undefined', () => {
+    const ctx = {};
+    const result = isApiRequest(ctx);
+    expect(result).toBe(false);
+  });
+});
```

Comment:
@BorysShulyak Do you think we can remove the `undefined` checks here for simplification? I feel like the first 2 tests which check the `content-api` value to be true or false is sufficient.

Reply:
@mattmilburn Removed the `undefined` checks. Thank you!